File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TreeViews_ViewMyStringRowHandlesRow.cpp

#include "TreeViews_ViewMyStringRowHandlesRow.h"
#include "TreeViews_ViewMyStringRowHandlesRow.h"
#include <string>

namespace widgetassertions
{
  std::string TreeViews_ViewMyStringRowHandlesRow::getValuesLabelText( ) 
  {
    return this->ValuesLabelText;
  }
  std::string TreeViews_ViewMyStringRowHandlesRow::getRowHandle( ) 
  {
    return this->RowHandle;
  }
  int TreeViews_ViewMyStringRowHandlesRow::getRowDepth( ) 
  {
    return this->RowDepth;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TreeViews_ViewMyStringRowHandlesRow.cs
/// <filename>
///     TreeViews_ViewMyStringRowHandlesRow.cs
/// </filename>


namespace widgetassertions
{
    public class TreeViews_ViewMyStringRowHandlesRow
  {
        private string ValuesLabelText;
     
        public     virtual string getValuesLabelText()
    {
      return this.ValuesLabelText;
    }
     
        private string RowHandle;
     
        public     virtual string getRowHandle()
    {
      return this.RowHandle;
    }
     
        private int RowDepth;
     
        public     virtual int getRowDepth()
    {
      return this.RowDepth;
    }
  }
}

